wagtail-{{ cookiecutter.project_name_kebab }}/tox.ini

```diff
@@ -2,43 +2,51 @@
 skipsdist = True
 usedevelop = True
 
-envlist = python{3.7,3.8,3.9}-django{3.0,3.1,3.2,main}-wagtail{2.14,2.15,main}-{sqlite,postgres}
-
-[flake8]
-# E501: Line too long
-# W503: line break before binary operator (superseded by W504 line break after binary operator)
-ignore = E501,W503
-exclude = migrations,node_modules
+envlist =
+    python{3.7,3.8,3.9}-django{3.2}-wagtail{2.15,3.0,4.0}-{sqlite,postgres}
+    python{3.10}-django{3.2}-wagtail{2.15,3.0,4.0,main}-{sqlite,postgres}
+    python{3.7,3.8,3.9,3.10}-django{4.0}-wagtail{3.0,4.0,main}-{sqlite,postgres}
+    python{3.8,3.9,3.10}-django{4.1,main}-wagtail{4.0,main}-{sqlite,postgres}
```

Comment:
Python 3.11 has been released in the time between submitting this and someone reviewing it, and wagtail 4.1. Shall we get both tox and the workflow file testing on the latest versions, or just get this merged and then do another Pr bumping them. I don't want to let perfect be the enemy of the good too much here..

Reply:
added to the mix